### DISCUSSION TITLE
How to mark articles older than one day as read in the filter

### DISCUSSION THREAD

**Discussion by wcx** [FreshRSS/FreshRSS#6603] [Q&A]
Hello,how to mark articles before yesterday as read in the filter，I tried ‘date:P1D/’, ’date:/P1D’, ’date:/P-1D’，but it didn't work.Anyone can help me?Thank you!

**Comment by Alkarex** [comment]
Hello,
try with `pubdate:` instead. See example in point 6 of https://github.com/FreshRSS/FreshRSS/discussions/6101#discussioncomment-8479032
